docker-entrypoint.sh

```diff
@@ -1,24 +1,24 @@
 #!/bin/sh
 set -e
 
-if [ "${USE_PROXY_PROTOCOL}" -eq "1" ]; then
+if [ "${USE_PROXY_PROTOCOL:-0}" -eq "1" ]; then
   set -- -x "$@"
 fi
 
-if [ "${HAS_WEBSOCKET}" -eq "1" ]; then
+if [ "${HAS_WEBSOCKET:-0}" -eq "1" ]; then
   set -- -w localhost "$@"
 fi
 
 if [ -n "${WEBSOCKET_HOSTNAME}" ]; then
   set -- -w "${WEBSOCKET_HOSTNAME}" "$@"
 fi
 
-if [ ! -z "${SSH_HOSTNAME}" ]; then
+if [ -n "${SSH_HOSTNAME}" ]; then
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2236

`! -z` and `-n` are equivalent - note that the check[ on line 12](https://github.com/tmate-io/tmate-ssh-server/pull/96/files#diff-79738685a656fe6b25061bb14181442210b599f746faeaba408a2401de45038aR12) uses `-n`